### PR TITLE
Incremental Release notes

### PR DESCRIPTION
**Pull request by JosephSafouri**
This updates for V 0.2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,4 +37,26 @@ Praharsh Patel </br>
...
+### Bug Fixes
+<ul>
+    <li> <em>Image and tool bar are seprate, but previosly they were overlapping.</em> </li>
+    <li> <em>The background was defaulted to yellow but now is white.</em> </li>
+    <li> <em>The width of the images wasn't the correct size type, but now is.</em> </li>
+</ul>
+
+### Known Issues
+<ul>
+    <li> Image uploading only throws an error rather than promt the user to select a different file.</li>
```
Comment (jasondpham) [inline README.md]: typo here can you fix :)

File: README.md [hunk 2]
```diff
@@ -37,4 +37,26 @@ Praharsh Patel </br>
...
+### Features
+<ul>
+    <li> Added a button to allow drawing a straight line</li>
+    <li> Added a button to allow the user to select an image</li>
+    <li> Added ability to detect whether the file is an image type or not</li>
+</ul>
+
+### Bug Fixes
+<ul>
+    <li> <em>Image and tool bar are seprate, but previosly they were overlapping.</em> </li>
```
Comment (jasondpham) [inline README.md]: Can you not make these 3 points italicized

Comment (JosephSafouri) [reply README.md]: Yes, I have made the commit to change it to be normal text and not italicized.

File: README.md (hunk 1 above)
Comment (JosephSafouri) [reply README.md]: Thanks for catching that, I changed promt to prompt.

**Comment by praharsh2001** [comment]
Good Incremental Release Notes. LGTM

**Comment by sarahukani** [comment]
Typo: please change "previosly" to "previously" on line 50

**Comment by sarahukani** [comment]
(referring to line 51) 
The default image was a solid color but we fixed it so that it should be set to a default hand x-ray image now. Are you able to see this change?

**Pull request merged**